src/Tribe/Views/V2/Assets.php

```diff
@@ -316,6 +317,22 @@ public function register() {
 				'in_footer'    => false,
 			]
 		);
+
+		tribe_asset(
```

Comment:
Excuse my ignorance around tribe_asset(), but what is the behavior when the file is not located? Also, I have heard that tribe_asset is a bit more expensive than we’d like-I wonder if it’d be more performant to try to locate_stylesheet before invoking tribe_asset?

Reply:
If it's not found it doesn't load anything. Have to check the asset implementation but I feel that check must be part of the logic inside the function.

Reply:
I just dug into `tribe_asset()`. It most definitely does what you are saying, but after executing a bunch of things. Let's wrap this tribe_asset() call in an if where we check for the stylesheet first.

Reply:
Seems like we have that and other checks covered: https://github.com/moderntribe/tribe-common/blob/master/src/Tribe/Assets.php#L367

And, we wouldn't be adding more "cost", as it's how we currently do it: https://github.com/moderntribe/the-events-calendar/blob/master/src/Tribe/Assets.php#L316